applications/motion_estimator_surf_test.cpp

```diff
@@ -0,0 +1,121 @@
+/* 
+ *  Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2016, Natalnet Laboratory for Perceptual Robotics
+ *  All rights reserved.
+ *  Redistribution and use in source and binary forms, with or without modification, are permitted provided
+ *  that the following conditions are met:
+ *
+ *  1. Redistributions of source code must retain the above copyright notice, this list of conditions and
+ *     the following disclaimer.
+ *
+ *  2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and
+ *     the following disclaimer in the documentation and/or other materials provided with the distribution.
+ * 
+ *  3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or
+ *     promote products derived from this software without specific prior written permission.
+ * 
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, *  INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ *  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ *  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+ *  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+#include <cstdio>
+#include <cstdlib>
+#include <Eigen/Geometry>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <geometry.h>
+#include <rgbd_loader.h>
+#include <motion_estimator_surf.h>
+#include <reconstruction_visualizer.h>
+#include <surf_detector.h>
+using namespace std;
+
+
+int main(int argc, char **argv)
+{
+	
+	string index_file_name;
+	RGBDLoader loader;
+	Intrinsics intr(0);	
+	MotionEstimatorSURF motion_estimator(intr);
+	ReconstructionVisualizer visualizer;
+        cv::Mat frame, depth;
+	Eigen::Affine3f pose = Eigen::Affine3f::Identity();
+	Eigen::Affine3f trans = Eigen::Affine3f::Identity();
+	pcl::PointCloud<PointT>::Ptr prev_cloud(new pcl::PointCloud<PointT>);
+	pcl::PointCloud<PointT>::Ptr curr_cloud(new pcl::PointCloud<PointT>);
+	SurfDetector detector_surf(400);
+	if(argc != 2)
+	{
+		fprintf(stderr, "Usage: %s <index file>\n", argv[0]);
+		exit(0);
+	}
+
+	index_file_name = argv[1];
+	loader.processFile(index_file_name);
+	//Track points on each image
+
+        cv::Mat prev_frame;
+
+	for(int i = 0; i < loader.num_images_; i++)
```

Comment:
Change spacing and indentation

Reply:
The _applications/motion_estimator_surf_test.cpp_ file  was deleted.